### ISSUE TITLE
Cities and counties statics going dark 

### ISSUE THREAD

**Issue by mcteapot** [CSSEGISandData/COVID-19#1068]
I have noticed over the past few days all the information for cities and counties is being deleted and aggregated into regions or states. 

Cases are growing rapidly in regions and states but it is impossible to determine if cases are growing around your specific area. 

For large states like California, the number is rapidly growing but it is impossible to tell if it is growing in your local area. 

The only way to find out about new cases in your locations is now only through news articles that are behind paywalls or vague updates by media.

**Comment by ravenspy57** [comment]
Agreed, seems like major counties like Los Angeles and Cook County are not being updated properly.

**Comment by alterisian** [comment]
I agree that it would be great to have regional data for Spain. Even within Andalucia there is great variation. In Málaga City it shut down earlier than the rest of the region. So it would be great to illustrate that people's efforts are worth it, and show that the rate of growth of new cases, is either below the exponential curve, or eventually actually decreasing. 

Here is an example: https://twitter.com/AndaluciaJunta/status/1240607863220965376 
I have asked them for the source of the data.

**Comment by lijianathome** [comment]
Correct me but I believe most repositories have similar data sources.  Who can offer us permission to update the data please?

**Comment by mcteapot** [comment]
Looking at the Google Sheet that is updated on stats there seems to be lat and long of cases. Not really human-readable, but I guess you can code a distance check to see if inside a county or city boundary.  

https://docs.google.com/spreadsheets/d/1avGWWl1J19O_Zm0NGTGy2E-fOG05i4ljRfjl87P7FiA/edit?ts=5e5e9222#gid=1223604501

**Comment by sp1ff** [comment]
+1 to this-- the SF Bay Area has been dark the past few days, as well.

**Comment by Sleekery** [comment]
Yeah, starting 3/10/20, they're only doing the US by states. Province/State is smallest subcategory they go, and only for some countries (dependencies of some nations and provinces/states of US, China, Canada, Australia, I believe).

**Comment by mcteapot** [comment]
Looks like the maps are not being updated by country and cities, Thank you so much whoever made the changes! 

Lets we if we can now get it via a data feed as well.
